examples/maven-example/README.md

```diff
@@ -28,9 +28,9 @@ The following properties should be added to your environment or this example's [
 
 From the project directory, run the following:
 
-*Linux/Mac*: `./mvnw -s ./settings.xml spring-boot:run` or `./run.sh`
+*Linux/Mac*: `mvn -s ./settings.xml spring-boot:run` or `./run.sh`
 
-*Windows*: `mvnw -s .\settings.xml spring-boot:run` or `run.bat`
+*Windows*: `mvn -s .\settings.xml spring-boot:run` or `run.bat`
```

Comment:
- If you really want to remove the wrapper from the examples, address all of the work outlined in the first comment associated with this file.
- If you are okay with leaving the wrapper in with the example, accept the below suggestion.
```suggestion
*Windows*: `mvnw -s .\settings.xml spring-boot:run` or `run.bat`
```